FIS Package Update

pa-ulex opened the pull request:
This general update to the FIS package brings it up to date with the latest plugin versions, adds a mapbuilder setup and new TopSky maps.

The coordinates were taken from the German, Dutch, Danish, Swedish, Polish, Czech, Austrian, Swiss and French AIPs or NAV-Packages of VATGER.
The sector data was taken from ATCISS/VATGlasses.

pa-ulex committed:
Inital Commit + GNG Sync
Fixed AUP URL and added shortcuts
Added mapbuilder workflow
Initally started TopSky Maps

actions-user committed:
[AUTO] mapbuilder dedeeabc244d6d5df08ca256be728dcfa73e8cfb

pa-ulex committed:
Map Update 1
Updated Airspace, Runways and ECLs
Added Germany Border and VRPs initally

actions-user committed:
[AUTO] mapbuilder 6e19cfff7da2278b48f66c6bacfd4a532d430ca5

pa-ulex committed:
Map Update 3
Updated Areas, SSRcodes, VRP, Airspaces, FIRs, Settings and Sectors

actions-user committed:
[AUTO] mapbuilder 0ac8b840d0cdd94a4edf650d024be53d2c46e14e

pa-ulex committed:
Map Update 2
Updated Airspaces, Runways and ECLs

actions-user committed:
[AUTO] mapbuilder 61994187ffafa66bd66f2298eb0331333b966c13

pa-ulex committed:
Map Update 4
Updated Airspace Labels and Structure

actions-user committed:
[AUTO] mapbuilder 0625a1e64116abb26e46ee7dbb548d2fd3a719c0

pa-ulex committed:
Map Update 5
New Airspace Folder Structure and
added EDMM RMZ Runways

actions-user committed:
[AUTO] mapbuilder 8fdfddffbe47aad0fe892b6ccdbd9f916911bd21

pa-ulex committed:
Map Update 6
Completed VRPs and
began adding External FIS Sectors (WIP)

actions-user committed:
[AUTO] mapbuilder 4379234c5f2f24dddf723c3634cef2f518d6bb99

pa-ulex committed:
Map Update 7
Removed External FIS Sectors from maps (WIP for future)
Added Neighbour/Bordering FIRs
Corrected AUP Settings

actions-user committed:
[AUTO] mapbuilder 4cc8897b971597d6d0b53f7d363b172a290d9031

sergej-singer commented:
So, I'm going to be apparently the first one, who is going to leave anything here.

On itself, looks better than before. Still, I assume it's WIP, because here and there, there are things to improve.

- Colors: I see that, not all the colors were adjusted. I would suggest matching menus either with EDMM or EDGG profiles (i.e. it's currently mixed, you have old default grey chat, beige TopSky menu like in EDGG and gray background like in EDMM).
- TopSky GUI is too small.
- FIS border resolution is inconsistent with state borders from Topo Map.
- Some maps are unnecessarily divided into multiple ones (moreover, placed into different folders), like Helipads and Labels for Helipads, as well as Topo maps.
- Not sure why there are two folders for airspace.
- TMZ/RMZ should be on by default.
- Some maps don't work at all: ~~navaids~~, urban area maps.

There's probably something more to add, but at a quick glance, that's all I could find. Surely there must be some testing concluded before release.

pa-ulex committed:
Map Fix 1
Removed Sound within Profiles
Fixed colors to match EDGG colors
Made GUI Bigger
Default activate TMZ/RMZ
Removed unnessesary map folders
Added Urban Areas
Fix EDDM D CTR
Fix FIS Layer
Added TopSky View
(Non Relevant) Added further neighbour FIS freq.

actions-user committed:
[AUTO] mapbuilder 8d18f907b7148426b4d7e71f3d27110b425e6757

pa-ulex commented:
> FIS border resolution is inconsistent with state borders from Topo Map.

Should have fixed everything except for the FIS border. I will probably fix this within the next fixes as they are currently only copied from the already existing package.

hannesaltmann commented:
- I would love to see all those areas mentioned in ENR 5.5 implemented
- Display of glider areas would be nice
- irl the ED-Rs are grey
- I guess it's a good idea to have the coastline displayed by default. From what I saw this is always displayed irl (at least EBG North)
- The airspace labels are different irl (only numbers, see screenshot below). In general, I think these labels can be a bit smaller compared to the current state. 
- The urban areas can hardly be seen. Maybe slightly increase the contrast to the background. 
- Labels for some major cities could be useful for a general orientation
- Maybe consider implementing some lines to indicate different neighbouring sector (e.g. for distinction between Gdansk and Poznan Information)
- For Berlin I would prefer an airspace C variant which is visible at all times. Maybe only the HX sectors could be activated depending on the runway?
- Personally, I would not link the ECLs to any automatic activation. Manual activation is fine, as ECLs hardly have any relevance for FIS.

![Screenshot_20250607-173150](https://github.com/user-attachments/assets/9d60dcb2-5ae4-496d-8d92-74d9662572f2)

pa-ulex commented:
> * Maybe consider implementing some lines to indicate different neighbouring sector (e.g. for distinction between Gdansk and Poznan Information)

Already asked our neighbouring FIRs for sectors and frequencies, currently waiting for response.

Everything else mentioned above should be fixed with the next update but I'm planning to fix incorrect Maps/Settings first and then implementing new features.

Also thanks for the picture!

actions-user committed:
[AUTO] mapbuilder 7d9d9cebc7c07a92a54870ab82ccb5f781f5167a

pa-ulex commented:
The second update also includes a number of fixes. My to-do list now includes integrating ENR 5.5–7 onwards, and including the FIS sectors and borders as soon as I receive a message about them.

I also rewrote a few commits to make the schema better.

pa-ulex committed:
Map Fix 2
Changed TopSky Location
Added LF** aerodromes and Navaids
Added Navaid Tags
Added Platzsektor Jägerhaus
Added Urban/City Labels
Added Obstacles
Added PJE Dropping Zones
Changed Urban Colors
Added Berlin Geneal Airspace
Fixed altitude labels

actions-user committed:
[AUTO] mapbuilder 8074969d0033a1370d5a7b7f8cc23d08e4fc7b42

pa-ulex committed:
Map Fix 3
Added TopSky Radars
Added better Border
Added more Airspace QNHs
Added Basel Airspace
Corrected FIS Sectors DS and DM
Moved topo folders
Fixed wrong shortcut
Fixed some PJE spots
Fixed TopSkySSRcodes (DW -> SW)
Removed unnecessary lines
Added Glider Airfields

actions-user committed:
[AUTO] mapbuilder b211be66a0da34d7dd875d0840745857780dd058

pa-ulex committed:
Map Fix 4
Reworked General Folder Structure
Added/Corrected VCCS
Added FIR Border
Added/Finished ENR 5.5 Maps
Added MIL RAFIS Sectors
Added Offshore Heli Routes
Added more City Labels
Added various External Airspaces
Started adding Airspace Labels
Corrected Layers
Changed German Border
Changed QGIS Layers
Changed some PRF Settings
Changed Shortcuts
Updated Obstacles
Fixed various Lists
Moved Squwark Plugin
Removed Glider Airfields
Removed CCAMS
Deactivated CPDLC by Default
Updated ECLs

pa-ulex committed:
Map Fix 5
Adjusted Squawks
Updated TopSky Areas
Reworked Aerodromes
Reworked Navaids Text Positions
Fix ALT Shortcuts
Added TMZ Labels
Corrected Special Airspace locations (within TopSky Maps)
Removed Neighbour Sectors for now

actions-user committed:
[AUTO] mapbuilder 15c533fef69795b49893618a6f2a176ffa7dd58b

pa-ulex committed:
Map Fix 6
Add vLARA URL
Add External Airspaces (ELLX, ESMS, EKRN, LFSB, LOWS)
Add missing Aerobatic Sectors
Add TMZ Labels
Add D(CTR) Labels
Fix Navaid Labels
Updated MVA
Adjusted Labels
Adjusted GH Actions name
Adjusted FIS FIR Sector file name
Reworked Layers
Removed FAP from mapbuilder toml
Fix Autobahn Topo

pa-ulex committed:
Map Fix 7
Relocate Topsky Folder
Fix Squawk Plugin JSON
Adjuste Map Sorting
Update Obstacles
Update MVA
Small City Label Change
Change to Winter MVA
Remove ECLs
Update Workflow

actions-user committed:
[AUTO] mapbuilder 93be010af9a266623860b8a12221956a2948019d

pa-ulex committed:
Map Fix 8
Add new Mode S SSR Region
Rename SquawkIndicator Plugin
Fix numbering
Initially added D(CTR) dynamic activation
Correct Tags
Clarified Helipad usage
Fix typos
Remove OSN VOR

actions-user committed:
[AUTO] mapbuilder 379cc3f85a2d1694d4365f50d6d9f0119252c71b

hannesaltmann commented:
Drei Punkte die mir noch aufgefallen sind:
- Es ist weiterhin schwierig zu erkennen, wo die FIRs oder Sektoren unserer Nachbarn anfangen. Zumindest so ein kleiner Strich, der die FIR-Grenze darstellt, wäre super. Ich muss ja schon wissen, ob ich mit EKDK oder ESMM telefonieren muss. 
- An der Grenze zu unseren Nachbarn sind einige RNAV-Wegpunkte als Bedarfsmeldepunkte angegeben. Diese würde ich zumindest in einer Map unterbringen
- Vielleicht habe ich es auch übersehen, aber ich konnte keine Map finden, wo ich mir die Locations und ICAO Codes alles ADs schnell einblenden kann

Sonst wirklich super Pack!

hannesaltmann commented:
Und die ED-Rs und TRAs sollten so aktiviert werden, wie es auch in den normalen packs ist. Also bspw. TRAs und ED-Ds etc. nur per vLARA Aktivierung und die normalen ED-Rs und LANTA per AUP.

hannesaltmann commented:
Typo City Labels: Dresen --> Dresden

lucianoasmus commented:
- Color_Assumed=0,0,0 fehlt in der settings datei
- frequency plan list imo unnötig
- separator vor dem optext im non-detailed label
- aerodrome runways standardmäßig an
- toggle route draw in der uncontrolled list funktioniert nicht, ich empfehle ohne autohide, das funktioniert
- shortcuts wäre nice für
-Squawk map
-Quick-look (dieser standard topsky filter-aus-toggle)
-aerodromes
-Küste auf alt+k
- How about tower and lower radar frequency list? Dass man ein shortcut hittet und dann die frequenz ready hat wo die piloten für CTR crossing, IFR Pickup etc rufen können

Special request
- Das TopSky SSR setting zu nutzen, wie die münchener das haben, dass man auf '"assign squawk" bei einem vfr flugzeug klicken kann und dann der FIS squawk assigned wird

Sonst schon ganz nice, man merkt aber vom feeling, dass das auf alten Langen Pack versionen basiert.
